### ISSUE TITLE
Carthage Installation fails for usage via Checkouts

### ISSUE THREAD

**Issue by Dschee** [nicklockwood/iCarousel#782]
I'm using the most current release (1.8.3) and include the iCarousel files manually into my project from within the Carthage Checkout folder. But when checking out Carthage doesn't simply skip iCarousel when building, because there are several .xcproject files with shared schemes. And one of them (or more?) fails with this:

```
xcodebuild timed out while trying to read iCarouselChameleonDemo.xcodeproj 😭
```

**Comment by Jeehut** [comment]
I made an attempt to fix this, see https://github.com/nicklockwood/iCarousel/pull/783.
I've also included instructions to use my fork there until my PR is merged.

**Comment by Jeehut** [comment]
@nicklockwood Thanks for merging my PR so quickly, but it seems I posted it a little too early. According to the Carthage README running `carthage build --no-skip-current` within the framework directory is sufficient to test if building works correctly. That's what I tested (and that is working) before sending over my PR, but when I actually include the current master branch to a project via `github "nicklockwood/iCarousel" "master"` and try to build, I still get the above mentioned error.

It seems something is really wrong with the iCarouselChameleonDemo.xcodeproj when using the Xcode command line tools. So my new approach is to remove all example projects from this library (they can be placed into their own repo for example) and use it like that. If it works, I'll report. I'm not sure if you're OK with such a change, but if you're not we could also keep that version in its own branch (e.g. named "carthage") and add a hint to the README for all Carthage users.
